src/CMakeLists.txt

```diff
@@ -1799,9 +1799,14 @@ install(
 install(FILES start/libcp2k.h
         DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}")
 
+if(NOT CMAKE_INSTALL_Fortran_MODULES)
```

Comment:
I would use CP2K_INSTALL_Fortran_MODULES (or shorter version) instead of the CMAKE_ prefixed one. Note that this variable is not defined as such by cmake but it might be in the future. 

An dedicated option is also better than relying on the knowledge that `-DCP2K_INSTALL_Fortran_MODULES` can be defined. It is exactly the same problem than for "CP2K_DATA_DIR"

Reply:
Well, we did go with this in dbcsr in anticipation - see https://github.com/cp2k/dbcsr/pull/701/files